source/content/object-cache.md

```diff
@@ -467,7 +473,7 @@ $conf['cache_inc'] = 'sites/all/modules/cache_backport/cache.inc';
 
 ### You have requested a non-existent service
 
-The following error occurs when modifying configurations for the [Redis](https://www.drupal.org/project/redis) module before it has been enabled:
+The following error occurs when modifying configurations for the [Redis](https://www.drupal.org/project/object-cache) module before it has been enabled:
```

Comment:
broken link - the Drupal project is located at https://www.drupal.org/project/redis